### ISSUE TITLE
panic: reflect.Value.Interface: cannot return value obtained from unexported field or method

### ISSUE THREAD

**Issue by Lilja** [alexflint/go-arg#145]
New to go. I installed this library and tried to follow the examples. I would like to add a verbose flag to show some logs of the application. I can't seem to get this working...

```go
// Cli.go
package main

import (
  "github.com/alexflint/go-arg"
  "fmt"
)

var version = "0.1.0"

type cli struct {
  verbose string `help:"Verbose mode, logs stuff"`
}

func whatever() {
  var args cli
  arg.MustParse(&args)
  fmt.Println(args.verbose)
}
```

```go
// main.go
package main

func main() {
  whatever()
}
```

```
panic: reflect.Value.Interface: cannot return value obtained from unexported field or method

goroutine 1 [running]:
reflect.valueInterface(0x52ddc0, 0xc000048890, 0x1b8, 0x582d01, 0x52ddc0, 0x0)
        /home/lilja/.go/src/reflect/value.go:1027 +0x169
reflect.Value.Interface(...)
        /home/lilja/.go/src/reflect/value.go:1016
github.com/alexflint/go-arg.isZero(0x52ddc0, 0xc000048890, 0x1b8, 0x1)
        /home/lilja/.go/pkg/mod/github.com/alexflint/go-arg@v1.3.0/parse.go:757 +0x15a
github.com/alexflint/go-arg.NewParser(0x0, 0x0, 0xc00014bf18, 0x1, 0x1, 0x7f462e22a108, 0x10, 0xc000048890)
        /home/lilja/.go/pkg/mod/github.com/alexflint/go-arg@v1.3.0/parse.go:213 +0x5da
github.com/alexflint/go-arg.MustParse(0xc00014bf18, 0x1, 0x1, 0xc000032720)
        /home/lilja/.go/pkg/mod/github.com/alexflint/go-arg@v1.3.0/parse.go:82 +0x65
main.whatever()
        /home/lilja/code/smap/cli.go:16 +0x74
main.main()
        /home/lilja/code/smap/main.go:5 +0x25

```

**Comment by atc0005** [comment]
Contents of a modified `cli.go` file:

```golang
// Cli.go
package main

import (
	"fmt"
	"github.com/alexflint/go-arg"
)

var version = "0.1.0"

type cli struct {
	Verbose string `help:"Verbose mode, logs stuff"`
}

func whatever() {
	var args cli
	arg.MustParse(&args)
	fmt.Println(args.Verbose)
}
```

I ran it via:

```console
$ go run main.go cli.go --verbose "hello"
hello
```

The complaint was that the struct field wasn't exported, which is required for this package to work as expected. I changed it from `verbose` to `Verbose`.

Apologies for the brief response, hoping that it's still useful.

This package is great btw, I expect that you'll come to like it after getting to know it a bit more.

**Comment by alexflint** [comment]
Thank you for noting this Erik and thank you for posting the solution Adam.

Just so you know, this panic was fixed in #136, but that change hasn’t been
released as a version yet.
